TicketsDemo.EF/Repositories/RunRepository.cs

```diff
@@ -43,17 +43,15 @@ public List<Data.Entities.Run> GetRuns(DateTime start, DateTime end, int? traint
         {
             using (var ctx = new TicketsContext())
             {
-                return ctx.Runs.Where(r => r.Date > start && r.Date < end && (traintId == null || r.TrainId == traintId))
-                    .Include(x => x.Places)
-                    .ToList();
+                return ctx.Runs.Where(r => r.Date > start && r.Date < end && (traintId == null || r.TrainId == traintId)).ToList();
             }
         }
 
         public Data.Entities.Run GetRunDetails(int runId)
         {
             using (var ctx = new TicketsContext())
             {
-                return ctx.Runs.Include(r => r.Places)
+                return ctx.Runs.Include("Places")
```

Comment:
but... why?

If you really don't like lambdas - use nameof(Carriage.Places) instead of string literal.